src/components/Card/StandardCard/styles.scss

```diff
@@ -1,5 +1,6 @@
 @use '../../../theme/variables.scss' as vars;
```

Comment:
For this entire file, change the colors definitions, since the theme change implementation, we need to start using only carbon's css vars, for instance: `var(--bcgov-background-brand)`. Take a look at the Figma file to get the correct var value to use :)